src/dotty/tools/dotc/FromTasty.scala

```diff
@@ -64,6 +64,9 @@ object FromTasty extends Driver {
   }
 
   class ReadTastyTreesFromClasses extends FrontEnd {
+
+    override def isTyper = false
```

Comment:
True. But it does not matter for anyway. The change did not affect the fixes. I'll add a comment to explain better what isTyper is.